project/src/components/filter-item/filter-item.tsx

```diff
@@ -0,0 +1,26 @@
+import './style.css';
+import {useDispatch, useSelector} from 'react-redux';
+import {changeGenre} from '../../store/action';
+
+type FilterItemProps = {
+  genre: string;
+}
+
+function FilterItem(props: FilterItemProps): JSX.Element {
+  const {genre} = props;
+  const activeFilter = useSelector((state: { genre: string; }) => state.genre);
+  const dispatch = useDispatch();
+
+  return (
+    <li
+      className={activeFilter === genre ? 'catalog__genres-item catalog__genres-item--active' : 'catalog__genres-item'}
```

Comment:
Чтобы избежать дублирования класса, можно записать так:
```suggestion
      className={`catalog__genres-item ${activeFilter === genre ? 'catalog__genres-item--active' : ''}`}
```